app/models/vm_or_template/operations/configuration.rb

```diff
@@ -125,10 +125,19 @@ def raw_detach_volume(_volume_id)
 
   def detach_volume(volume_id, _device = nil)
     raise _("VM has no EMS, unable to detach volume") unless ext_management_system
-
```

Comment:
@jdenver23 this looks unrelated to you clone_volume changes can you remove?

Reply:
Yes, I'll get that done